app/helpers/codelist_helper.rb

```diff
@@ -17,11 +17,16 @@ def yaml_to_objects(entity:, type:, with_empty_item: true)
     objects
   end
 
-  def yaml_to_objects_with_description(entity:, type:)
+  def yaml_to_objects_with_description(entity:, type:, code_displayed_in_name: false)
```

Comment:
Given that all(?) the IATI yaml files have codes this seems relevant as a generic change to this method. I notice it's taking on a fair few responsibilities though so one for us to keep an eye on.